src/Films/Films.css

```diff
@@ -0,0 +1,12 @@
+.cardsContainer {
```

Comment:
some simple styling so that we can kinda see what we are working with. i like that.